### ISSUE TITLE
`python3.7` raises `flake8` warning

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#93]
It is a bug in `flake8`:
- https://github.com/PyCQA/pycodestyle/issues/728

We currently allow `python3.7` build to fail.

```
=============================== warnings summary ===============================
tests/test_visitors/test_wrong_class/test_base_class.py::FLAKE8
  /home/travis/virtualenv/python3.7.0/lib/python3.7/site-packages/pycodestyle.py:113: FutureWarning: Possible nested set at position 1
    EXTRANEOUS_WHITESPACE_REGEX = re.compile(r'[[({] | []}),;:]')
-- Docs: https://docs.pytest.org/en/latest/warnings.html
=================== 1514 passed, 1 warnings in 27.96 seconds ===================
```

**Comment by sobolevn** [comment]
We will just ignore this warning for now.
